api/src/main/java/net/md_5/bungee/api/plugin/PluginManager.java

```diff
@@ -75,53 +73,46 @@ public PluginManager(ProxyServer proxy)
      * @param plugin the plugin owning this command
      * @param command the command to register
      */
-    public void registerCommand(Plugin plugin, Command command)
-    {
-        commandMap.put( command.getName().toLowerCase( Locale.ROOT ), command );
-        for ( String alias : command.getAliases() )
-        {
-            commandMap.put( alias.toLowerCase( Locale.ROOT ), command );
+    public void registerCommand(Plugin plugin, Command command) {
+        commandMap.put(command.getName().toLowerCase(Locale.ROOT), command);
+        for (String alias : command.getAliases()) {
+            commandMap.put(alias.toLowerCase(Locale.ROOT), command);
         }
-        commandsByPlugin.put( plugin, command );
+        commandsByPlugin.put(plugin, command);
     }
 
     /**
      * Unregister a command so it will no longer be executed.
      *
      * @param command the command to unregister
      */
-    public void unregisterCommand(Command command)
-    {
-        while ( commandMap.values().remove( command ) );
-        commandsByPlugin.values().remove( command );
+    public void unregisterCommand(Command command) {
+        while (commandMap.values().remove(command));
+        commandsByPlugin.values().remove(command);
     }
 
     /**
      * Unregister all commands owned by a {@link Plugin}
      *
      * @param plugin the plugin to register the commands of
      */
-    public void unregisterCommands(Plugin plugin)
-    {
-        for ( Iterator<Command> it = commandsByPlugin.get( plugin ).iterator(); it.hasNext(); )
-        {
+    public void unregisterCommands(Plugin plugin) {
+        for (Iterator<Command> it = commandsByPlugin.get(plugin.iterator(); it.hasNext();) {
```

Comment:
commandsByPlugin.get(plugin).iterator()
Also, have you checked the coding style used before submitting this pull request?

Reply:
No, I have not. Where can I find that?